priv/proto3/google/protobuf/any.proto

```diff
@@ -1,6 +1,3 @@
-// This file is imported from protobuf 3.12.3
-//
-
 // Protocol Buffers - Google's data interchange format
 // Copyright 2008 Google Inc.  All rights reserved.
 // https://developers.google.com/protocol-buffers/
```

Comment:
It would be nice for future references if you could add a comment like the one removed, that says what version of protobuf the file came from (same for the rest of the updates)

Reply:
Done